From e09fe44d241eed6cb27b8d10a6e16b9080e9b625 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Mon, 19 Mar 2012 10:07:30 +0800
Subject: [PATCH] adc: fix kbuild warning

drivers/adc/plat/Kconfig:10:warning: defaults for choice values not supported
---
 drivers/adc/plat/Kconfig | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/drivers/adc/plat/Kconfig b/drivers/adc/plat/Kconfig
index cbdd09c1d15f..ac0fd467e568 100644
--- a/drivers/adc/plat/Kconfig
+++ b/drivers/adc/plat/Kconfig
@@ -4,13 +4,12 @@
 
 choice
 	prompt "ADC hardware drivers"
+	default ADC_RK29 if ARCH_RK29
+	default ADC_RK30 if ARCH_RK30
 
 config ADC_NULL
 	bool "NULL"
-	default y
-	help
-		null
-	
+
 config ADC_RK28
 	bool "RK28 adc interface"
         depends on ARCH_RK2818 || ARCH_RK2808 || ARCH_RK2816
-- 
2.35.3

